src/pages/blog.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Blog</title>
+  <link rel="stylesheet" href="../styles/output.css">
+</head>
+
+<body>
+  <header class="header">
+    <img class="logo" src="../assets/icons/Finsweet black logo.svg" alt="FinSweet black logo">
+    <nav class="header__nav-links">
+      <a href="#" class="link">Home</a>
+      <a href="./pages/product.html" class="link">Product</a>
+      <a href="#" class="link">Pricing</a>
+      <a href="#" class="link">About us</a>
+      <a href="./pages/blog.html" class="link">Blog</a>
+      <a href="#" class="link">Contact</a>
+      <button type="button" class="button button--primary">Free trial</button>
+    </nav>
+  </header>
+  <section class="pt-24 flex flex-col gap-16">
+    <div class="section-text section-text--centered">
+      <h1 class="section-text__heading section-text__heading--big">Read our latest blogs
+      </h1>
+      <p class="description description--big">Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy
+        eirmod tempor
+        invidunt ut labore et
+        dolore
+        magna aliquyam erat.</p>
+    </div>
+  </section>
+  <section class="halved-section halved-section--pushed-down">
+    <div class="section-text">
+      <h4 class="section-text__label">Trending Post</h4>
+      <h2 class="section-text__heading max-w-[552px]">New invoicing features to help you get paid faster</h2>
+      <p class="description desciption--why-us-section">Over the past few months, we’ve added several new features to
```

Comment:
`desciption--why-us-section` is included without it's default Element (desciption).